src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -669,13 +669,22 @@ void MenuGame::initLua(Lua::LuaContext& mLua)
 
     mLua.writeVariable("s_setPulseInc",
         [this](float mValue) { styleData.pulseIncrement = mValue; });
+    
+    mLua.writeVariable("s_setPulseIncrement",
+        [this](float mValue) { styleData.pulseIncrement = mValue; });
```

Comment:
This should already be automatically generated by:

```cpp
sdVar("PulseIncrement", &StyleData::pulseIncrement);
```

Reply:
This isn't the case at all for any of the lsVar or sdVar functions. I did this because it's an alias for PulseInc and I wanted to get all the nooks and crannies. sdVar doesn't add any functions to mLua, and this is proven because a Discord member reported this and shows mLua reports errors at style functions, despite them being valid functions.

Reply:
I failed to realize this was in `MenuGame.cpp`, sorry.